# You can find control table of Dynamixel on emanual (https://emanual.robotis.com/docs/en/dxl/x/xl330-m288/)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
# This file is compatible with Dynamixel using protocol 2.0
tilt:
  ID: 1
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 0
  Profile_Velocity: 20
  Position_P_Gain: 4000
  Position_I_Gain: 0
pan:
  ID: 2
  Return_Delay_Time: 0
  Operating_Mode: 3
  Profile_Acceleration: 0
  Profile_Velocity: 20
  Position_P_Gain: 4000
  Position_I_Gain: 0
